src/utils.spec.tsx

```diff
@@ -3,13 +3,31 @@ import queryString from 'query-string';
 
 import {
   formatFilesize,
+  getCodeLineAnchorGetter,
   getLanguageFromMimeType,
   getLocalizedString,
   getPathFromQueryString,
   nl2br,
   sanitizeHTML,
 } from './utils';
-import { createFakeHistory, createFakeLocation } from './test-helpers';
+import {
+  createFakeCompareInfo,
+  createFakeHistory,
+  createFakeLocation,
+} from './test-helpers';
+
+// If we don't create the mock for ForwardComparisonMap here it doesn't work!
+const mockGetterFromForwardComparisonMap = jest.fn();
+const mockCreateCodeLineAnchorGetter = jest
+  .fn()
+  .mockReturnValue(mockGetterFromForwardComparisonMap);
+jest.mock('./pages/Compare/utils', () => {
```

Comment:
This will be very confusing for the future person who has to debug it. It suggests to me that the class is being instantiated incorrectly or that you need dependency injection or maybe something else.

Reply:
I've removed all the mocking.